apps/mobile-alphabet/src/app/Components/Menu-detail.presenter.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Image, Text, View } from 'react-native';
+import config from './Config.json';
+import { AlphabetData } from './Menu';
+
+export function MenuDetialScreen() {
+    const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
+
+    const [selectedLetterSequenceNumber, setSelectedLetterSequenceNumber] = useState<number>(1);
+
+    useEffect(() => {
+        //TODO cache this after fetching it once
```

Comment:
If you haven't already, make a story for this and link it here in a comment. Also link the story in the code comment.